docs/advanced_topics/api/v2/configuration.md

```diff
@@ -45,6 +45,23 @@ Wagtail provides three endpoint classes you can use:
 -   Documents {class}`wagtail.documents.api.v2.views.DocumentsAPIViewSet`
 
 You can subclass any of these endpoint classes to customize their functionality.
+
+For example:
+```python
+
+# In this case if you need to customize APIViewSet by setting a desired renderer class
+
+from rest_framework.renderers import JSONRenderer
+
+...
```

Comment:
```suggestion
# ...
```

Minor suggestion - adding a comment start here makes it easier to copy/paste this snippet as is.